chatapplication/src/index.js

```diff
@@ -2,13 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
+import LoginPage from './Pages/LoginPage';
 import reportWebVitals from './reportWebVitals';
+import {BrowserRouter, Routes, Route} from 'react-router-dom'
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+    <BrowserRouter>
+        <Routes>
+          <Route exact path="/" element={<LoginPage />} />
+          <Route exact path="registration" element={<App />}/>
```

Comment:
Why is the App rendered under registration?
It should not. We need the Register component here instead.